gnomad_qc/v3/subset/subset.py

```diff
@@ -181,62 +211,112 @@ def main(args):
-                                                    subset_AF=hl.float32(mt.subset_callstats.AF[1])))
-        mt = mt.drop('subset_callstats')
-        header_dict['info'].update({"subset_AC": {"Description": "Alternate Allele count in subset post-filtering"} , 
-                                    "subset_AN": {"Description": "Ref and Alternate allele number in subset post-filtering"},
-                                    "subset_AF": {"Description": "Alternate Allele frequency in subset post-filtering"}})
+        mt = mt.annotate_rows(subset_callstats_raw=hl.agg.call_stats(mt.GT,mt.alleles))     
+        mt = mt.annotate_rows(info=mt.info.annotate(subset_AC_raw=mt.subset_callstats.AC[1],
+                                                    subset_AN_raw=mt.subset_callstats.AN,
+                                                    subset_AF_raw=hl.float32(mt.subset_callstats.AF[1])))
+        mt = mt.drop('subset_callstats_raw')
+        mt = annotate_adj(mt)
+        mt = mt.annotate_rows(subset_callstats_adj=hl.agg.filter(mt.adj, hl.agg.call_stats(mt.GT,mt.alleles)))
+        mt = mt.annotate_rows(info=mt.info.annotate(subset_AC_adj=mt.subset_callstats_adj.AC[1],
+                                                    subset_AN_adj=mt.subset_callstats_adj.AN,
+                                                    subset_AF_adj=hl.float32(mt.subset_callstats_adj.AF[1])))
+        mt = mt.drop('subset_callstats_adj')
+        header_dict['info'].update({
+            "subset_AC_raw": 
+                {
+                "Number": "A",
+                "Description": "Alternate allele count in subset before filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
+                },
+            "subset_AN_raw": 
+                {
+                "Number": "1",
+                "Description": "Total number of alleles in subset before filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
+                },
+            "subset_AF_raw": {
+                "Number": "A",
+                "Description": "Alternate allele frequency in subset before filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
+                },
+            "subset_AC_adj": {
+                "Number": "A",
+                "Description": "Alternate allele count in subset post-filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
+                },
+             "subset_AN_adj": {
+                "Number": "1",
+                "Description": "Total number of alleles in subset post-filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)"
+                },
+            "subset_AF_adj": {
+                "Number": "A",
+                "Description": "Alternate allele frequency in subset post-filtering of low-confidence genotypes (GQ < 20; DP < 10; and AB < 0.2 for het calls)",
```

Comment:
Above and below you have `after filtering` here you have `post-filtering`, either is fine, but maybe make the same